### PR TITLE
Gui widget fixes1

### PR DESCRIPTION
**Pull request by shenjackyuanjie**
fix some typing in `event`
and fix `TextEntry` background

image

### CODE REVIEW

**Comment by shenjackyuanjie** [comment]
image

oh, I forgot to reinstall it, it's fine

File: pyglet/gui/widgets.py [hunk 1]
```diff
@@ -445,7 +445,7 @@ def __init__(self, text: str,
                 Optional parent group of text entry widget.
         """
         self._doc = pyglet.text.document.UnformattedDocument(text)
-        self._doc.set_style(0, len(self._doc.text), dict(color=text_color))
```
Comment (benmoran56) [inline pyglet/gui/widgets.py:448]: Doesn't the linter prefer `dict(...)` instead of `{...}`?

**Comment by benmoran56** [comment]
Closing because this work is also in the other PR.

File: pyglet/gui/widgets.py (hunk 1 above)
Comment (shenjackyuanjie) [reply pyglet/gui/widgets.py:448]: it prefer `{...}` better

**Pull request closed without merging**